src/main/java/ch/njol/skript/expressions/ExprLootItems.java

```diff
@@ -0,0 +1,107 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.loot.LootContext;
+import org.bukkit.loot.LootTable;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
+
+@Name("Loot Table Items")
+@Description({
+	"Returns the items of a loot table using a loot context.",
+	"Note that loot contexts only require the killer and looted entity to be set if the loot table is under the entities category.",
+	"The loot context only requires the location if the loot table is not in the entities category, eg. blocks, chests."
+})
+@Examples({
+	"set {_items::*} to loot items of the loot table \"minecraft:chests/simple_dungeon\" with loot context {_context}",
+	"# this will set {_items::*} to the items that would be dropped from the simple dungeon loot table with the given loot context",
+	"",
+	"give player loot items of entity's loot table with loot context {_context}",
+	"# this will give the player the items that the entity would drop with the given loot context"
+})
+@Since("INSERT VERSION")
+public class ExprLootItems extends SimpleExpression<ItemStack> {
+
+	static {
+		Skript.registerExpression(ExprLootItems.class, ItemStack.class, ExpressionType.COMBINED,
+			"[the] loot[ ]item[s] of %loottables% (with|using) [the] [[loot] context] %lootcontext%",
+			"[the] %loottables%'[s] loot[ ]item[s] (with|using) [the] [[loot] context] %lootcontext%"
```

Comment:
```suggestion
			"[the] [loot[ ]]item[s] of %loottables% (with|using) [the] [[loot] context] %lootcontext%",
			"[the] %loottables%'[s] [loot[ ]]item[s] (with|using) [the] [[loot] context] %lootcontext%"
```

I think having loot optional here would be better, as you specify its for a loot table anyway (the syntax in the suggestion might be wrong idk)